Cargo.toml

```diff
@@ -34,12 +34,9 @@ micromath = { version = "1.1.0", default-features = false }
 embedded-graphics-core = { path = "core", version = "0.3.2"}
 byteorder = { version = "1.3.4", default-features = false }
 
-# criterion is not listed as a dev-dependency to work around a problem with no_std compatibility
-# and because dev-dependencies can't be optional
-criterion = { version = "=0.3.4", optional = true }
-
```

Comment:
So nice we can finally get rid of this ugly hack.

Reply:
Yeah! Much cleaner now :)